gtsam/nonlinear/tests/testExpression.cpp

```diff
@@ -170,29 +181,29 @@ Point3_ p_cam(x, &Pose3::transformTo, p);
 }
 
 /* ************************************************************************* */
-// Check that creating an expression to double compiles
+// Check that creating an expression to double compiles.
 TEST(Expression, BinaryToDouble) {
   using namespace binary;
   Double_ p_cam(doubleF, x, p);
 }
 
 /* ************************************************************************* */
-// keys
+// Check keys of an expression created from class method.
 TEST(Expression, BinaryKeys) {
   set<Key> expected = list_of(1)(2);
   EXPECT(expected == binary::p_cam.keys());
 }
 
 /* ************************************************************************* */
-// dimensions
+// Check dimensions by calling `dims` method.
 TEST(Expression, BinaryDimensions) {
   map<Key, int> actual, expected = map_list_of<Key, int>(1, 6)(2, 3);
   binary::p_cam.dims(actual);
   EXPECT(actual == expected);
 }
 
 /* ************************************************************************* */
-// dimensions
+// Check dimensions by calling `traceSize` method.
```

Comment:
This actually checks the size of the “trace record”, which is where an execution trace is stored.